README.md

```diff
@@ -20,6 +20,8 @@ Repo: [github.com/NebraLtd/hm-pktfwd](https://github.com/NebraLtd/hm-pktfwd)
 
 This container is responsible for configuring packet forwarder's region and starts the radio module.
 
+Packet-forwarder container requires port [1680](https://github.com/helium/gateway-rs/blob/d6e140fc8f102d2e1008ddf6d58cef32c4f60392/src/settings.rs#L18) exposed as default for connecting to gatewayrs over localhost (127.0.0.1).
```

Comment:
i don't think this is correct.

gateway-rs / miner needs to have port 1680 open. Packet-forwarder does not need any

Reply:
You're right. I've removed it from the file but forgot removing from readme.